tools/install-source/OpenTKSourceMangler.cs

```diff
@@ -20,14 +23,27 @@ public class OpenTKSourceMangler : IPathMangler
 		/// <value>The install dir.</value>
 		public string InstallDir { get; set;}
 
-		public string GetSourcePath (string path) => path;
+		public string GetSourcePath (string path)
+		{
+			bool iosFramework = true;
+			if (path.StartsWith(OpenTKSourcePath, StringComparison.Ordinal)) 
+				return path;
+			// we are dealing with a package build
+			var index = path.IndexOf (iOSFramework, StringComparison.Ordinal);
+			if (index < 0) {// we are dealing with mac sources
+				iosFramework = false;
+				index = path.IndexOf(MacFramework, StringComparison.Ordinal);
+			}
+			path = path.Remove (0, index + ((iosFramework)?iOSFramework.Length : MacFramework.Length)); // + length framework
+			return Path.Combine (OpenTKSourcePath, path);
+		}
 
 		public string GetTargetPath (string path)
 		{
 			var relativePath = path.Substring (OpenTKSourcePath.Length);
 			if (relativePath.StartsWith ("/", StringComparison.Ordinal))
 				relativePath = relativePath.Remove (0, 1);
-			var target = Path.Combine (InstallDir, "src", relativePath);
+			var target = Path.Combine (InstallDir, "src", (InstallDir.Contains("Xamarin.iOS")?"Xamarin.iOS":"Xamarin.Mac"), relativePath);
```

Comment:
Coding style: space before opening parenthesis, and between between parts of expressions.